main.py

```diff
@@ -138,6 +138,9 @@ def main(input_path, output_path):
                 "Successfully parsed images: {}".format(successful_files))
             logging.info(
                 "Files with unsupported file extensions: {}".format(other_files))
+            if no_output_directory:
+                for file in os.listdir('{}/converted-text'.format(input_path)):
```

Comment:
You have implemented this for each file instead it should be a global flag i.e., if the output_path was not passed in arguments print output of every file on console and do not save it to any file